presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix hanging join operator when spill is enabled and the probe side finishes before the hash builder starts.
+* Add file based password authenticator plugin.
+* Add new spilling strategy to spill when the user + system + revocable memory on a node exceeds max_total_memory_per_node.  This can be enabled by setting ``experimental.spiller.task-spilling-strategy`` to ``PER_QUERY_MEMORY_LIMIT``.
+* Add support for logging formatted prepared query. This could be enabled by setting session property ``log_formatted_query_enabled`` to ``true``.
+* Add support for passthrough Elasticsearch queries.
+* Add support for returning partial results for the queries by setting `partial_results_enabled` session property. Additionally `partial_results_max_execution_time_multiplier`, `partial_results_completion_ratio_threshold`  session properties can be set to configure the max execution time multiplier and minimum completion ratio threshold for the queries.
+* Add automatic query retry functionality for transient failures. This can be enabled by setting per-query-retry-limit to a non-zero integer to indicate the per query retry count.
+* Store dictionary elements in Segmented Slice.
```

Comment:
What does this mean ? How does this impact a user?

Reply:
@arunthirupathi @highker Could you add more details here?

Reply:
ORC dictionary writer stores the contents of the dictionary in segmented slices, instead of one contiguous slice.

Reply:
@highker mentioned that we don't need release notes for this commit, so I'm removing this.